kolibri/plugins/user/assets/src/views/SignInPage/index.vue

```diff
@@ -484,6 +487,10 @@
           this.$refs.username.focus();
         }
       },
+      handleUsernameInput() {
```

Comment:
I added this so that the `invalidText` under the textbox would appear immediately, since the only other way would be to blur the textbox now that the form and next button are disabled..

Reply:
alright make sense, thank you for the help!